admin/admin-static-pages.php

```diff
@@ -25,16 +30,34 @@ public function __construct( &$polylang ) {
 		parent::__construct( $polylang );
 
 		$this->links = &$polylang->links;
+		$this->post  = &$polylang->model->post;
+
+		add_action( 'after_setup_theme', array( $this, 'init_page_hooks' ), 1000 ); // Hook late to allow other sources to filter `'pll_get_post_types'` easily.
+	}
+
+	/**
+	 * Launches hooks related to the `page` post type.
+	 *
+	 * @since 3.7
+	 *
+	 * @return void
+	 */
+	public function init_page_hooks(): void {
+		if ( ! in_array( 'page', $this->post->get_translated_object_types(), true ) ) {
```

Comment:
Can't we use `Post_Types::is_translated_post_type()` instead? Seems more appropriate to me.

Reply:
🤦 
Done in [dbf052c](https://github.com/polylang/polylang/pull/1616/commits/dbf052c2f48737f7320ba080ae68ed8b01d6ea2d).
Note: I could have used `PLL_Translated_Post::is_translated()` but it felt better suited to use `Post_Types` as you suggested.